FontSize.cpp

```diff
@@ -0,0 +1,9 @@
+#include "FontCommon.h"
+#include "FontSize.h"
+
+
+FontSize::operator unsigned int(void) const
+{
+    const unsigned int dpi = 125;
+    return (dpi * pt) / 72; //(72 * MAX_EM) / (125 * pt);
```

Comment:
72 is a magic value, but is what is defined as a 1-inch font

Reply:
Maybe just make that a constant or macro to improve readability